website/docs/zkvm/quickstart.md

```diff
@@ -42,14 +42,27 @@ Don't worry -- each `TODO` comes with instructions for what you need to do, and
 
 ## 4. Run your project, locally or remotely
 
-The readme for your project includes instructions for how to run it, both locally or remotely.
-You can build your example and run the prover locally with:
+The readme for your project includes instructions for how to run it, both locally or remotely. You can also get a better developer experience and smoother iteration on your code by using the developer mode.
+
+### Running your project
+
+To run your project locally, going through the full proof generation process, run:
 
 ```bash
 cargo run --release
 ```
 
-> **Congratulations!** <br/>_That's all it takes to build and run a minimal RISC Zero application._
+#### Tip: Leveraging Developer Mode
+
```

Comment:
```suggestion
Depending on the length of execution, the `cargo run --release` command can take a long time to run. This command spends the majority of its time generating valid proofs. While it is important to produce proofs, waiting for while running code can slow down iteration during development. Rather than waiting for proofs on each iteration, developers can disable proof generation for development purposes and re-enable proving when they are done with each iteration.
```

I think we should add something like this to frame the problem that developers might encounter and maybe edit the beginning of the next paragraph for a smoother transition

Reply:
Thank you for the suggestion, Erik!